.azure-pipelines.yml

```diff
@@ -25,6 +25,11 @@ trigger:
 - triggers/safari_stable
 - triggers/safari_preview
 
+# Uncomment this to enable safaridriver diagnostics. The logs won't appear in
+# `./wpt run` output but will be uploaded as a separate artifact.
+#variables:
+#  safaridriver_diagnose: true
```

Comment:
I'm allergic to commented-out code; it tends to just bit-rot. Can we make this enabled but `false`, with instructions to change it to `true`?

Reply:
I ended up here because it seems like false is parsed as the string "false" in Azure Pipelines. It is a boolean false in various YAML parses I've tried, but maybe putting the value into a variable turns it into a string. Not sure. I'll try some things to see if it's possible to uncomment this.

Reply:
I made it work with `eq(variables['safaridriver_diagnose'], true)`. It does still seem from the logs when the "Enable safaridriver diagnostics" is skipped (`Expanded: eq('false', True)`) that it's a string, but `true` does work.

Looking at [the docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops) I still suspect the reason it works is a bit yuck though: the 2nd argument (boolean true) will be converted to a string, and then the string comparison is case-insensitive.

But I'm going to keep it like that, I looked at built-in tasks and found https://github.com/Azure/powershell/blob/b479e3884630075377f13b18fbc37af0a33cc20f/src/main.ts#L14 with a defaults-to-false variable, and see that it also does something involving strings.

Reply:
Thanks for doing this digging!